### ISSUE TITLE
[Feature Request]: Is this project dead?

### ISSUE THREAD

**Issue by techie2000** [devcontainers-contrib/features#628]
### Requested Feature

Firstly, apologies for filing under [Feature Request] but I do not see Discussions open for this project.

I see there has been no commits for ~6months at the time of writing so am wondering if this project is dead/no longer accepting contributions?

**Comment by erikspigle-payroc** [comment]
I'm curious as well. The ansible feature is failing to build and causing problems with another project from another repo where ansible is being referenced as a feature build for a dev container.

**Comment by flokoe** [comment]
Hi @danielbraun89 

After more than six months of inactivity in the repo, it seems reasonable to ask if this repo is still being maintained?

It appears that you own the organization, or at least are the person with the most commits/merge permissions. If I am mistaken, I apologize for the inconvenience.

I am particularly interested in #613 to be merged. @benklett already analyzed the problem and provided a solution. Is there anything that is currently blocking the progression?

I understand that the work here is probably done on a voluntary basis. So if you currently have no bandwidth available, that's fine, but I would kindly ask you to select some more individuals with permissions to merge to spread the risk of blocking the development of this repository. If there is anything else we can do, please let us know.

Best regards

**Comment by evilhamsterman** [comment]
I think this entire project and related repositories are unmaintained. Despite the name I don't believe this is a project that is officially associated with the upstream [Devcontainers Project](https://containers.dev/). It is run by one guy @danielbraun89 and it looks like he's had limited online activity for several months. It may or may not be related, but according to his Linkdin he's based in Israel and his GitHub activity essentially stopped at the beginning of October 2023, about the time of the latest Israel-Palestine conflict.

**Comment by eitsupi** [comment]
For context: devcontainers/devcontainers.github.io#168

**Comment by flokoe** [comment]
> It may or may not be related, but according to his Linkdin he's based in Israel and his GitHub activity essentially stopped at the beginning of October 2023, about the time of the latest Israel-Palestine conflict.

Yeah, this thought occurred to me too. Hopefully, this is not the case.

However, as this seems to be indeed a personal playground and the author seems to have no intention to collaborate more closely with the Devcontainer Project (which is fine) and even deleting issues/discussions about it (which is a red flag to me and leaves a bad taste in your mouth) I am no longer using this project. Thanks to @eitsupi for providing more context!

For those who are interested: I am currently evaluating [devbox](https://www.jetify.com/devbox/docs/) as an alternative. This is a wrapper around the Nix package manager (Simple JSON file, no need to learn Nix language). The isolation mechanism is of course different, but you got the entire nixpgs Repo at your disposal and less risk of packages not being maintained.

**Comment by evilhamsterman** [comment]
I opened a ticket with the devcontainers project to remove or alert about this project. They decided to remove it from their features and templates indexes. 

I think it was a great group of features, hopefully now it will encourage people to create replacements. Since the code is all MIT licensed anyone could certainly fork/copy anything here as a base
